prow/test/integration/integration-test.sh

```diff
@@ -31,6 +31,12 @@ retry() {
 }
 
 main() {
+  # Remove retries after
+  # https://github.com/bazelbuild/bazel/issues/12599
+  if [ -n "${BAZEL_FETCH_PLEASE:-}" ]; then
```

Comment:
Specify this in the integration job spec in this PR too?

Reply:
sure, meat to test it out in a separate PR but since I need to run it manually any way, will validate in this PR